.travis.yml

```diff
@@ -113,3 +113,10 @@ matrix:
         - curl -S -f https://codecov.io/bash -o codecov
         - chmod +x codecov
         - ./codecov -x "llvm-cov gcov" -Z
+    # Clang format build
+    - os: linux
+      env: BUILDTYPE=debug
+      node_js: 4
```

Comment:
This looks good. I feel like an improvement would be to strip way even more details to focus this job exclusively on formatting. It could do `language: generic` and override `install:` to just run the `setup.sh` and not the `make` step - then this would run even faster and would not need the custom `env`.